editor/modes/visual-editor/style.scss

```diff
@@ -65,6 +65,34 @@
 		transition: 0.2s outline;
 	}
 
+	&.is-showing-mobile-controls {
+		.editor-visual-editor__block-controls > div {
+			display: none;
+		}
+		.editor-visual-editor__mobile-tools {
+			align-items: baseline;
+			display: flex;
+			width: 100%;
+			.editor-block-mover,
+			.editor-block-settings-menu {
+				display: block;
+				display: flex;
+				position: static;
+				.components-button {
+					margin: 0 5px;
+				}
+			}
+			.editor-block-settings-menu {
+				z-index: 99;
```

Comment:
Should use `z-index` function.

https://github.com/WordPress/gutenberg/blob/master/editor/assets/stylesheets/_z-index.scss